platform/mbed_stats.h

```diff
@@ -144,6 +147,10 @@ typedef struct {
     uint32_t cpu_id;                    /**< CPUID register data (Cortex-M only supported) */
     mbed_compiler_id_t compiler_id;     /**< Compiler ID \ref mbed_compiler_id_t */
     uint32_t compiler_version;          /**< Compiler version */
+    uint32_t ram_start[MBED_MAX_MEM_REGIONS];/**< Start address of RAM memory */
```

Comment:
Very very minor nit in the comment, should it be something like "Start addresses" since this field is an array of RAM start fields?

Reply:
Fixed